gnosis/safe-react
# Add third party cookies banner warning

## What it solves
Resolves https://github.com/gnosis/safe-react/issues/2501

## How this PR fixes it
We are adding a warning banner if the user has the third-party-cookies disabled. If that's the case some safe-apps can completely or partially fail.

For that we are using an HTML file loaded trough an iframe (injected dynamically) for detecting and communicating safe-web (postMessage) if the cookies are really disabled in its own execution context.

## How to test it
1) Disable Third party cookies in chrome settings (chrome://settings/cookies)
2) Open any app
3) The banner should show for 10 seconds
4) Test safe-web is not having issues when the iframe url returns 404 (Safe to deploy to prod without the redirection configured)

## ⚠️ Before merging ⚠️ 
After QA tested, change src URL for the dynamic iframe to the prod one
